style.css

```diff
@@ -8,29 +8,32 @@ body {
   margin: 0;
 }
 
-.centering-outer-container {
-  position: absolute;
-  display: table;
+.page-container {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: 100%;
-  height: 100%;
+  height: 100vh;
 }
 
-.centering-inner-container {
-  display: table-cell;
-  vertical-align: middle;
-  text-align: center;
-}
-
-.centering-content {
-  display: inline-block;
-}
-
-#content {
-  background-color: #fefefe;
-  max-width: 400px;
-  height: 400px;
-  padding: 20px;
+.content-box {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  justify-content: center;
```

Comment:
These other flex directives also need vendor prefixes. You can use this app to get all the prefixes: https://autoprefixer.github.io/